tests/dist/fixtures.h

```diff
@@ -14,16 +14,11 @@ class DistTestsFixture
   , public ConfTestFixture
   , public SnapshotTestFixture
 {
-  protected:
-    std::set<std::string> otherHosts;
```

Comment:
I noticed in passing that this isn't actually used anywhere so I removed it as part of this PR. Happy to move elsewhere but it's just 5 lines in this one file.